tests/test_text_extraction.py

```diff
@@ -129,7 +130,24 @@ def test_layout_mode_font_class_to_dict():
         "font_dictionary": {},
         "space_width": 8,
         "subtype": "foo",
-        "width_map": {},
+        "font_descriptor": FontDescriptor(
+            name="Unknown",
+            family="Unknown",
+            weight="Unknown",
+            ascent=700.0,
+            descent=-200.0,
+            cap_height=600.0,
+            x_height=500.0,
+            italic_angle=0.0,
+            flags=32,
+            bbox=(
+                     -100.0,
```

Comment:
This indentation looks odd. I guess we can dedent lines 144 to 148 by one level.